src/overlays/actors/ovl_Bg_Jya_1flift/z_bg_jya_1flift.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_bg_jya_1flift.c
+ * Overlay: ovl_Bg_Jya_1flift
+ * Description: Shortcut Elevator used in the vanilla version of the Spirit Temple
```

Comment:
I feel like "Spirit Temple Elevator (1st floor)" is probably a better description. For now it makes sense to mention that its for vanilla, but I think that can be implied. Especially when in the end we'll be supporting more vanilla versions than mq, it feels weird to single it out

Reply:
The change was Roman's suggestion.  I had it as Shortcut Elevator (Spirit temple) Unused in master quest.  Or something like that